tests/FlexTest.php

```diff
@@ -32,6 +32,25 @@
 
 class FlexTest extends TestCase
 {
+    /**
+    * Call protected/private method of a class.
+    * taken from Juan Tremino's fantastic tutorial
+    * https://jtreminio.com/2013/03/unit-testing-tutorial-part-3-testing-protected-private-methods-coverage-reports-and-crap/
+    *
+    * @param object &$object    Instantiated object that we will run method on.
+    * @param string $methodName Method name to call
+    * @param array  $parameters Array of parameters to pass into method.
+    *
+    * @return mixed Method return.
+    */
+    public function invokeMethod(&$object, $methodName, array $parameters = array())
```

Comment:
why a reference ? This does not make any sense here. We are not in a PHP 4 world anymore. There is no need to pass objects by reference.

Reply:
fixed. (sorry was due to copy & paste)